src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -31,6 +33,10 @@ public OverseasEntitySubmissionDtoValidator(EntityDtoValidator entityDtoValidato
 
     public Errors validateFull(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
 
+        if (UtilsValidators.isNotNull(overseasEntitySubmissionDto.getEntityName(), OverseasEntitySubmissionDto.ENTITY_NAME_FIELD, errors, loggingContext)) {
```

Comment:
we'll also need to update the validatePartial method to validate the EntityName block if it is present. This will happen when we have save and resume enabled and we submit the json to the api at the end of the entity name screen

Reply:
Updated partial validation along with another test